File: Iniciante/URI 1044.r
input = file('stdin', 'r')

line = strsplit(readLines(input, n=1), " ")
a = as.integer(line[[1]][1])
b = as.integer(line[[1]][2])

if (a %% b == 0 || b %% a == 0) {
    write(sprintf("Sao Multiplos"), '')
} else { 
    write(sprintf("Nao sao Multiplos"), '')
}

File: Iniciante/URI 1044.php
<?php
    $line = explode(" ", trim(fgets(STDIN)));
    $a = floatval($line[0]);
    $b = floatval($line[1]);

    if ($b % $a != 0 && $a % $b != 0) {
        echo "Nao sao Multiplos\n";
    } else {
        echo "Sao Multiplos\n";
    }
?>